noxfile.py

```diff
@@ -1,45 +1,51 @@
 # -*- coding: utf-8 -*-
-'''
+"""
 noxfile
 ~~~~~~~
 
 Nox configuration script
-'''
+"""
 # pylint: disable=resource-leakage
 
 # Import Python libs
-from __future__ import absolute_import, unicode_literals, print_function
-import os
-import sys
+from __future__ import absolute_import, print_function, unicode_literals
+
+import datetime
 import glob
 import json
+import os
 import pprint
 import shutil
+import sys
 import tempfile
-import datetime
-
-if __name__ == '__main__':
-    sys.stderr.write('Do not execute this file directly. Use nox instead, it will know how to handle this file\n')
-    sys.stderr.flush()
-    exit(1)
 
 # Import 3rd-party libs
 import nox
 from nox.command import CommandFailed
 
+if __name__ == "__main__":
+    sys.stderr.write(
+        "Do not execute this file directly. Use nox instead, it will know how to handle this file\n"
+    )
+    sys.stderr.flush()
+    exit(1)
+
```

Comment:
The idea was to write to stderr even before trying to import nox.
We can wrap this code block with [fmt: on/off](https://black.readthedocs.io/en/stable/the_black_code_style.html#the-black-code-style).

Reply:
Can we move this back up?